axelor-production/src/main/resources/views/ManufOrder.xml

```diff
@@ -114,6 +114,7 @@
 			<field name="prodProcess.outsourcing" colSpan="3"/>
 			<field name="product" canEdit="false" required="true" readonlyIf="statusSelect &gt; 1 || !product.isModel" form-view="product-form" grid-view="product-grid" onSelect="action-manuf-order-attrs-filter-product"/>
 			<field name="qty" readonlyIf="statusSelect &gt; 1 || isPermanent" validIf="isPermanent || qty > 0"/>
+			<field name="unit" readonly="true"/>
```

Comment:
Not displaying beside qty. Change colspan of qty and unit to display beside qty.